libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -524,10 +549,12 @@ void AP_Baro::init(void)
 {
     init_done = true;
 
-    // ensure that there isn't a previous ground temperature saved
-    if (!is_zero(_user_ground_temperature)) {
```

Comment:
we seem to have lost this reset of ground temp?

Reply:
Yes I removed it because if you want it at 15, you want it at 15 almost all the time.  But if you want it to be automatically calculated/estimated for you as is the case with 99% of users, then you leave it equal to zero.